modules/scrub/scrub.go

```diff
@@ -16,62 +16,66 @@ var JSON_FILES_TO_SCRUB = [...]string{
 	"status.json",
 }
 
-func ScrubJsonFiles(localPath string) bool {
-	success := false
+var JSON_FIELDS_TO_SCRUB = [...]string{
+	"translators",
+	"contributors",
+	"checking_entity",
+}
+
+func ScrubJsonFiles(localPath string) error {
 	for _, fileName := range JSON_FILES_TO_SCRUB {
-		success = ScrubJsonFile(localPath, fileName) || success
+		if err := ScrubJsonFile(localPath, fileName); err != nil {
+			return err
+		}
 	}
-	return success
+	return nil
 }
 
-func ScrubJsonFile(localPath, fileName string) bool {
+func ScrubJsonFile(localPath, fileName string) error {
 	jsonPath := path.Join(localPath, fileName)
 
 	var jsonData interface{}
 	if fileContent, err := ioutil.ReadFile(jsonPath); err != nil {
-		return false
+		return nil
```

Comment:
Didn't want it to have to deal with the oddity of not being able to even read the file, but I guess I should. Yet then the user will still not really know why since these error messages are not flashed on the web page. Will make it return err.